### ISSUE TITLE
After updating Azure CLI is broken: ImportError: cannot import name 'parse_resource_id'

### ISSUE THREAD

**Issue by pjanowski** [Azure/azure-cli#11357]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**
I had an older versioin of azure cli and decide to update by running  Invoke-WebRequest -Uri https://aka.ms/installazurecliwindows -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet' in Powershell. Installation ran without incident. When I type "az" in powershell I now get

```
PS C:\Users\NN> az
The system cannot find the path specified.
The system cannot find the path specified.
Traceback (most recent call last):
  File "C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-umfcm_n2\azure-cli\azure\cli\__main__.py", line 33, in <module>
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-umfcm_n2\azure-cli-core\azure\cli\core\__init__.py", line 554, in get_default_cli
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-umfcm_n2\azure-cli-core\azure\cli\core\__init__.py", line 61, in __init__
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-umfcm_n2\azure-cli-core\azure\cli\core\commands\arm.py", line 182, in register_ids_argument      
ImportError: cannot import name 'parse_resource_id'

```

I tried uninstalling and reinstalling as well as removing my conda/python/pip installs on Windows thinking there might be a conflict there. 

I also have Azure CLI under WSL and works fine there.

**To Reproduce**
Install the msi and run any  "az" command

**Expected behavior**
Should have gotten the az help text.

**Environment summary**
Powershell with windows 1909 (OS Build 18363.476)

**Additional context**

**Comment by fengzhou-msft** [comment]
Thanks for reporting the issue.

**Comment by yonzhan** [comment]
add to S163.

**Comment by fengzhou-msft** [comment]
From the error trace, it looks like the CLI code from some pip-install directory is used? Can you run `get-command az` to see which az command is actually used?

**Comment by fengzhou-msft** [comment]
I cannot reproduce this issue with azure-cli 2.0.78 on Powershell. Please start PowerShell as administrator when running the installation command if you haven't done so.

**Comment by pjanowski** [comment]
`PS C:\Users\paulano> get-command az

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Application     az.cmd                                             0.0.0.0    C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin\az.cmd`

Running power shell as admin produces the same error.

**Comment by fengzhou-msft** [comment]
This command result looks fine. 

`az.cmd` is using a python executable inside msi package. The trace to call `C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-umfcm_n2\azure-cli\azure\cli\__main__.py` really seems weird to me. It should use `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\Lib\site-packages\azure\cli\__main__.pyc`.

Can you run `& 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe'` in Powershell and then check the module path in python prompt:
`>>>import azure.cli`
`>>>print(azure.cli.__file__)`

**Comment by pjanowski** [comment]
Here you go:
```
PS C:\WINDOWS\system32> & 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe'
Python 3.6.6 (v3.6.6:4cf1f54eb7, Jun 27 2018, 02:47:15) [MSC v.1900 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import azure.cli
>>> print(azure.cli.__file__)
C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\lib\site-packages\azure\cli\__init__.pyc
>>> 
```

**Comment by fengzhou-msft** [comment]
This is good. OK, one more try. Please run `az --debug` to print detailed info for the error.

**Comment by pjanowski** [comment]
This is in administrator PowerShell

```
PS C:\WINDOWS\system32> az --debug                                                                                      The system cannot find the path specified.
The system cannot find the path specified.
Traceback (most recent call last):
  File "C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-umfcm_n2\azure-cli\azure\cli\__main__.py", line 33, in <module>
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-umfcm_n2\azure-cli-core\azure\cli\core\__init__.py", line 554, in get_default_cli
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-umfcm_n2\azure-cli-core\azure\cli\core\__init__.py", line 61, in __init__
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-umfcm_n2\azure-cli-core\azure\cli\core\commands\arm.py", line 182, in register_ids_argument
ImportError: cannot import name 'parse_resource_id'
```

**Comment by fengzhou-msft** [comment]
This issue was fixed by manually remove the files under `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2` and reinstall azure-cli. The cause was that a reinstall of azure-cli package didn't remove the old `msrestazure` package automatically and the old package was invoked. 

The previous installed azure-cli may be too old. The install mechanism has changed after that version. I have verified for recent azure-cli packages, a new dependency package will replace the old one.
